### PR TITLE
:art: Add constant prefix to DiabloUI

### PR DESCRIPTION
**Pull request by AJenbo**

### CODE REVIEW

**Review by Jmgr — COMMENTED** [review]
Not a big fan of this one. Why does it matter that it is a constant?

File: Source/DiabloUI/selhero.cpp [hunk 1]
```diff
@@ -482,7 +482,7 @@ const char *SelheroGenerateName(HeroClass heroClass)
 
 	int iRand = rand() % 10;
 
-	return KNames[static_cast<std::size_t>(heroClass) % 6][iRand];
+	return kKNames[static_cast<std::size_t>(heroClass) % 6][iRand];
```
Comment (thebigMuh) [inline Source/DiabloUI/selhero.cpp:485]: Uiui, just one k short of KKK 😲

**Review by thebigMuh — APPROVED** [review]

**Comment by Jmgr** [comment]
> Not a big fan of this one. Why does it matter that it is a constant?

I would also keep the first upper case letter for types, not variables, constants or functions, but that's more a style question.

**Comment by AJenbo** [comment]
> > Not a big fan of this one. Why does it matter that it is a constant?
> 
> I would also keep the first upper case letter for types, not variables, constants or functions, but that's more a style question.

Using it for globals only, lets solve it by getting rid of them :D

File: Source/DiabloUI/selhero.cpp [hunk 2]
```diff
@@ -32,7 +32,7 @@ namespace {
 std::size_t selhero_SaveCount = 0;
 _uiheroinfo selhero_heros[MAX_CHARACTERS];
 _uiheroinfo selhero_heroInfo;
-const size_t KMaxViewportItems = 6;
+const size_t kKMaxViewportItems = 6;
```
Comment (julealgon) [inline Source/DiabloUI/selhero.cpp:35]: Wasn't the previous `K` here already intended to be the hungarian notation one?

File: Source/DiabloUI/selhero.cpp [hunk 3]
```diff
@@ -399,7 +399,7 @@ void SelheroLoadSelect(int value)
 
 const char *SelheroGenerateName(HeroClass heroClass)
 {
-	static const char *const KNames[6][10] = {
+	static const char *const kKNames[6][10] = {
```
Comment (julealgon) [inline Source/DiabloUI/selhero.cpp:402]: Same here.

**Pull request closed without merging**